src/gtest/test_foundersreward.cpp

```diff
@@ -81,21 +83,24 @@ TEST(founders_reward_test, create_testnet_2of3multisig) {
 
     pWallet->Flush(true);
 
-    ECC_Stop();
+    //ECC_Stop(); this is called on the main class of gtest
 }
-#endif
 
 
-#if 0 // Disabling all these tests for now until all the latest horizen commits have been integrated then will re-enable and fix
+
 
 // Utility method to check the number of unique addresses from height 1 to maxHeight
 void checkNumberOfUniqueAddresses(int nUnique) {
 
-    int maxHeight = Params().GetConsensus().GetLastFoundersRewardBlockHeight();
+    int maxHeight = Params().GetConsensus().GetLastCommunityRewardBlockHeight();
     printf("maxHeight = %d\n",maxHeight);
     std::set<std::string> addresses;
     for (int i = 1; i <= maxHeight; i++) {
-        addresses.insert(Params().GetFoundersRewardAddressAtHeight(i));
+    	if(Params().GetCommunityFundAddressAtHeight(i, Fork::CommunityFundType::FOUNDATION).size()>0){
+    		auto result=addresses.insert(Params().GetCommunityFundAddressAtHeight(i, Fork::CommunityFundType::FOUNDATION));
+
+    	}
+
```

Comment:
Please conform the rest of the file and substitute tab with 4 spaces.
Also please eliminate newline on line 101